sqlg-postgres-parent/sqlg-postgres-dialect/src/main/java/org/umlg/sqlg/sql/dialect/PostgresDialect.java

```diff
@@ -3301,6 +3304,20 @@ public String getFullTextQueryText(FullText fullText, String column) {
         return "to_tsvector('" + fullText.getConfiguration() + "', " + leftHand + ") @@ " + toQuery + "('" + fullText.getConfiguration() + "',?)";
     }
 
+    @Override
+    public String getArrayContainsQueryText(String column, ArrayContains<?> predicate) {
+        List<String> predicateValues =
```

Comment:
Should you use a set to prevent duplicates?

Reply:
That would be a good optimization, but I will do it in the predicate class itself.